src/DETHRACE/CMakeLists.txt

```
@@ -195,8 +219,16 @@ endif()
 
 if (DETHRACE_INSTALL)
     install(TARGETS dethrace
-        RUNTIME DESTINATION "."
+        BUNDLE DESTINATION . COMPONENT Runtime
+        RUNTIME DESTINATION bin COMPONENT Runtime
```

```suggestion
        RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}" COMPONENT Runtime
```
And do `include(GNUInstallDirs)` somewhere before this point.

---

This also means the windows zip will have `dethrace.exe` in a bin directory.
So perhaps we want to configure with `-DCMAKE_INSTALL_BINDIR=.` on Windows,
or do the following before `include(GNUInstallDirs)`:
```cmake
if(WIN32)
  set(CMAKE_INSTALL_BINDIR "." CACHE PATH "Binary installation directory")
endif()
include(GNUInstallDirs)
```

Also, make sure to install the extra runtime dlls (SDL2.dll) and the pdb (SDL2.pdb) to the same directory in [the lines below](https://github.com/dethrace-labs/dethrace/blob/673244bb8c19cdd3041bfd69e0891c504f58757a/src/DETHRACE/CMakeLists.txt#L196-L212)

---

How about set RUNTIME DESTINATION “.” as it was before maybe?

---

That's fine!
If somebody ever packages this, he can move it himself :)